devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy

```diff
@@ -70,11 +70,11 @@ def parseAttrMessage(description) {
 		map = getButtonEvent(descMap)
 	} else if(descMap?.clusterInt == 0xFC80) {
 		def buttonNumber
-		buttonNumber = Integer.valueOf(descMap?.command[1].toInteger()) + 1
+		buttonNumber = Integer.valueOf(descMap?.command[1].toInteger())
```

Comment:
What values are sent for each of the 4 button presses? Is it 0, 1, 2, 3 or 1, 2, 3, 4?

Reply:
0,1,2,3

0 sent to main button, 1,2,3 sent to child buttons